aten/src/ATen/native/cpu/TensorCompareKernel.cpp

```diff
@@ -299,8 +299,10 @@ static void isin_default_kernel_cpu(
   // Since test elements is not an input of the TensorIterator, type promotion
   // must be done manually.
   ScalarType common_type = at::result_type(elements, test_elements);
-  Tensor test_elements_flat = test_elements.to(common_type).ravel();
```

Comment:
`ravel` does not necessarily return a contiguous tensor. See #https://github.com/pytorch/pytorch/issues/70657

In anycase, we don't need a contiguous tensor.